app/controllers/email_templates_controller.rb

```diff
@@ -46,11 +46,8 @@ def email_template_params
     params.require(:email_template).permit(:subject, :body, :recipients)
   end
 
-  def search_template_params
```

Comment:
Hi i didnt change anything in this :/ I might have pressed delete button while closing it . Sorry. Can u tell me how to undo it :(